rhino/src/main/java/org/mozilla/javascript/BuiltInSlot.java

```diff
@@ -0,0 +1,108 @@
+package org.mozilla.javascript;
+
+import java.io.Serializable;
+
+public class BuiltInSlot<T extends ScriptableObject> extends Slot {
```

Comment:
As we get more an more such special classes (which is good) i think we have to try to make at least the intention as clear as possible. Therefore l like to vote for some class comment that helps in the future to decide if we can use this approach in other situations.

Reply:
Yeah, that’s definitely something I should clean up.